.github/workflows/docker-deployment.yml

```diff
@@ -16,7 +16,8 @@ jobs:
   push_to_regsitry:
```

Comment:
This looks like an existing typo (regsitry instead of registry) that does not seem to affect operation of this workflow.